src/parachain/fee.ts

```diff
@@ -55,10 +61,24 @@ export class DefaultFeeAPI implements FeeAPI {
 
     async getGriefingCollateral<C extends CollateralUnit>(
         amount: MonetaryAmount<Currency<BitcoinUnit>, BitcoinUnit>,
-        griefingCollateralRate: Big,
-        collateralCurrency: Currency<C>
+        collateralCurrency: Currency<C>,
+        type: GriefingCollateralType
```

Comment:
What is the benefit of have one function? IMO this would be cleaner with `getIssueGriefing` and `getReplaceGriefing`.

Reply:
Those functions do exist, but they're each exposed in their respective API. I added this function to parallelize two calls that were previously made in sequence